src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertToJsonCommand.cs

```diff
@@ -100,6 +100,16 @@ protected override void BeginProcessing()
         /// </summary>
         protected override void ProcessRecord()
         {
+            if (InternalTestHooks.ActivateSleepForStoppingTest)
+            {
+                // We'll wait the verbose stream marker in tests and then call Stop().
+                // The cmdlet is very fast and likely to end before the test sees the marker
+                // so we have to slow down it by Sleep(50).
+                // We can not sleep for a long time because Stop() doesn't work on sleeping runspace.
+                WriteVerbose("ConvertTo-Json started");
```

Comment:
what are you trying to achieve with the WriteVerbose? I'm not sure there's much value here and I would rather reduce the chatty output.

Reply:
The verbose message here is used to tell the test that the cmdlet has started so that it can issue the stop.  Otherwise there's a window where stop is issued before the cmdlet under test has started.

Reply:
Yes, we need an event to start the stopping.